target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-haze.dts

```diff
@@ -303,3 +310,41 @@
 
 	qcom,ath11k-calibration-variant = "prpl-Haze";
 };
+
+&blsp1_i2c3{
+	pinctrl-0 = <&i2c_3_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	led-controller@30 {
+		compatible = "ti,lp5562";
+		reg = <0x30>;
+		clock-mode = /bits/ 8 <2>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		status_red: chan@0 {
```

Comment:
Nitpick, there seems to be a convention to name the aliases in the following way

```suggestion
		led_system_red: chan@0 {
```